src/screens/IndexScreen.js

```diff
@@ -145,6 +148,14 @@ export const IndexScreen = ({ navigation, route }) => {
       [QUERY_TYPES.EVENT_RECORDS]: showEventsFilter,
       [QUERY_TYPES.NEWS_ITEMS]: showNewsFilter
     }[query];
+  const htmlContentName =
+    (query === QUERY_TYPES.POINTS_OF_INTEREST && poiListIntro?.[queryVariables.category]) || '';
+
+  const { data: htmlContent } = useStaticContent({
+    name: htmlContentName,
+    type: 'html',
+    refreshTimeKey: `${query}-${queryVariables.category}`
+  });
```

Comment:
this is the index screen, which is very much too complex already. with this, everytime accessing the index screen the hook is called and performs something. we need to avoid such cases with high priority! please add a `skip` here, if `htmlContentName` is not defined to nut run the hook and perform anything.

Reply:
Yes, you are right. We used it for the previous feature. I'm adding it here.
https://github.com/smart-village-solutions/smart-village-app-app/commit/a30ae59e770597a7b078e8ba7e44919e29c990c6